sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -78,9 +78,9 @@ public string Default()
 
         [HttpGet]
         [Route("/outgoing-sampleapp")]
-        public string Default()
+        public string Default2()
```

Comment:
Is there a better name here other than `Default2`?

Reply:
Changed it to OutgoingSampleApp